projects/planner/src/app/shared/serialization.model.ts

```diff
@@ -7,7 +7,7 @@ import { ViewState } from './views.model';
 
 export interface AppPreferences extends AppPreferencesDto {
     states: AppStates;
-    // quizAnswers?: Record<string, QuizAnswerStats>;
+    quizAnswers?: Record<string, QuizAnswerStats>;

```

Comment:
That is the issue, why it is commented this needs to load some default value.

Reply:
Use Map instead of Record and implement custom serialization of the stored values.